lib/rsgem/cli/commands/new.rb

```diff
@@ -24,6 +24,9 @@ class New < Dry::CLI::Command
         ]
 
         def call(**options)
+          puts Colors.colorize("Creating a new gem #{options[:gem_name]}", :green)
+          puts Colors.colorize("\tpassed in ci option = #{options[:ci]}", :green)
+          puts Colors.colorize("\tpassed in bundler option = #{options[:ci]}", :green) unless options[:bundler].nil?
```

Comment:
I'd rather leave the output information outside the CLI definition. The CLI related objects should only care about defining the valid options/arguments and mapping them into a method call of the gem, like we do in here for the new command. 

What do you think if instead we only update (and maybe colorize) these type of messages:
- https://github.com/rootstrap/rsgem/blob/master/lib/rsgem/tasks/create_gem.rb#L10
- https://github.com/rootstrap/rsgem/blob/master/lib/rsgem/tasks/clean_gemfile.rb#L21
- https://github.com/rootstrap/rsgem/blob/master/lib/rsgem/tasks/ignore_gemfile_lock.rb#L14

maybe any `puts` call in task objects in general (?)

Reply:
I thought about this after I submitted the PR, I agree that tasks should do the I/O and not this CLI object. I'll refactor this a bit.